site/content/en/docs/start/_index.md

```diff
@@ -525,6 +525,11 @@ You can also make your life easier by adding the following to your shell config:
 ```shell
 alias kubectl="minikube kubectl --"
 ```
+for windows :
```

Comment:
```suggestion
For Windows:
```

Reply:
this works for Windows with CMD 

for windows with powershell 

function kubectl { minikube kubectl -- $args }